extensions/capability-negotiation.md

```diff
@@ -131,6 +131,14 @@ with several subcommands.  The command added is named `CAP`.  `CAP` takes a sing
 required subcommand, optionally followed by a single parameter. Each subcommand defines any
 further parameters.
 
+The format of a client-sent `CAP` message is:
+
+    CAP <subcommand> [<parameters>]
```

Comment:
Maybe `[<parameters>...]` would make it clearer that these are multiple fields, not a single one?

Reply:
Good idea